src/components/Snackbar/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Snackbar as MuiSnackbar, Slide } from '@material-ui/core';
+import Icon from '@components/Icon';
+import { Props } from './types';
+
+
+/**
+ * <Snackbar {...props} /> Notification that pops up to alert the end user
+ * Visibility controlled by isOpen prop
+ */
+const Snackbar = ({
+    action = '',
+    className,
+    color = 'primary',
+    duration = 0,
+    message,
+    onClose,
+    iconName,
+    isOpen,
+    position = 'top-right',
+    slideDirection = 'left',
+    ...otherProps
+}: Props): React.ReactElement | null => {
+
+
+    /* Allows the component to be uncontrolled by end user */
+    const [uncontrolledIsOpen, setUncontrolledIsOpen] = React.useState(false);
+
+
+    /* Converts a string position 'top-right' to { vertical: 'top', horizontal: 'right' } */
+    const getAnchor = (): any => {
+
+        const anchorArray = position.split('-');
+
+        const [vertical, horizontal] = anchorArray;
+
+        return {
+            vertical,
+            horizontal,
+        };
+    };
+
+
+    React.useEffect(() => {
+
+        if (typeof isOpen === 'undefined') setUncontrolledIsOpen(true);
+
+    }, []);
+
+
+    const closeUncontrolled = (event?: React.SyntheticEvent, reason?: string): void => {
+
+        if (event) event.stopPropagation();
+        if (reason === 'clickaway') return;
+        if (typeof isOpen === 'undefined') setUncontrolledIsOpen(false);
+
+    };
+
+
+    const snackbarClasses = clsx(
```

Comment:
The classes don't have to be added this way, but the material UI components have a `classes` prop where you can add additional classes in as well.  https://material-ui.com/api/snackbar/#css

Reply:
Will fix.